ckanext/qdes_schema/helpers.py

```diff
@@ -84,7 +84,8 @@ def search_term_definition(terms, search_string):
         types = PackageRelationship.get_forward_types()
 
         nature_of_relationship = []
-        for term in get_action('get_vocabulary_service_terms')({}, 'nature-of-relationship'):
+        vocabulary_service_name = field.get('vocabulary_service_name', 'dataset-nature-of-relationship')
```

Comment:
This is hardcoded to only work with the dataset `dataset-nature-of-relationship`.
It needs to be updated to use the field property `vocabulary_service_name` to retrieve either `dataset-nature-of-relationship` or `dataservice-nature-of-relationship`

Reply:
this was more as failsafe option, but it wont need to be done as this